v0/destinations/ProfitWell/utils.js

```diff
@@ -0,0 +1,49 @@
+const get = require("get-value");
+const { removeUndefinedAndNullValues } = require("../../util");
+
+const createSubscriptionPayload = message => {
```

Comment:
we should use the `constructPayload` util function for these kind of payload mapping.